fvtest/porttest/si.cpp

```diff
@@ -796,6 +796,106 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 	reportTestExit(OMRPORTLIB, testName);
 }
 
+	/**
+	 *
+	 * Test omrsysinfo_test_sysinfo_set_limit and omrsysinfo_test_sysinfo_get_limit
+	 * with resourceID OMRPORT_RESOURCE_FILE_DESCRIPTORS
+	 *
+	 */
+	TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_FILE_DESCRIPTORS)
+	{
+		OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+		const char *testName = "omrsysinfo_test_sysinfo_set_limit_FILE_DESCRIPTORS";
+		intptr_t rc = -1;
+		uint64_t originalCurLimit = 0;
+		uint64_t originalMaxLimit = 0;
+		uint64_t currentLimit = 0;
+		const uint64_t descriptorLimit = 256;
+
+		reportTestEntry(OMRPORTLIB, testName);
+
+		/* save original soft limit */
+		omrsysinfo_get_limit(OMRPORT_RESOURCE_FILE_DESCRIPTORS, &originalCurLimit);
```

Comment:
Shouldn't these get_limit's check the return code?  If getLimit fails (ie: on an unsupported platform), the rest of the test shouldn't be executed

Reply:
Sounds reasonable to check it on the first call.  Would you like me to fix the other tests (which don't check the return code either)?

Reply:
Personally, I'd fix the calls to get_limit in this test in this PR and leave the others until such a time as the code is naturally being touched